parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -41,16 +41,17 @@ installDeps() {
         BLOBFUSE2_VERSION="2.2.0"
     fi
 
-    # Currently skipping blobfuse install on 24.04 since packages are not available upstream, remove this once they are
-    if [[ "${OSVERSION}" != "24.04" ]]; then
-      pkg_list+=(blobfuse2=${BLOBFUSE2_VERSION})
+    # Sync with Andy Zhang on if blofuse2 for 22.04 can be bumped to 2.3.0 to maintain parity, right now its 2.2.1
```

Comment:
blobfuse 2.3.0 works well, I have upgraded to 2.3.0 in blob csi driver upstream project, thx

Reply:
thanks Andy, I updated it for 22.04 as well!